scripts/scr_add_man/scr_add_man.gml

```diff
@@ -56,48 +107,75 @@ function scr_add_man(man_role, target_company, spawn_exp, spawn_name, corruption
-				obj_ini.gear[target_company][good] = "Storm Shield";
+				obj_ini.wep2[target_company][good] = "";
+				obj_ini.armour[target_company][good] = "Light Power Armour";
+				obj_ini.gear[target_company][good] = "Combat Shield";
 				obj_ini.experience[target_company][good] = 10;
 				obj_ini.race[target_company][good] = 4;
 				unit = new TTRPG_stats("inquisition", target_company, good, "inquisition_crusader");
 				break;
+			/* TODO - add capability for hirelings to have psychic powers and related stats
+			case "Sanctioned Psyker":
+				obj_ini.wep1[target_company][good] = "Force Staff"; TODO - consider adding regular human version
+				obj_ini.wep2[target_company][good] = "Laspistol";
+				obj_ini.armuor[target_company][good] = "Power Armour"; TODO - consider adding imperial armour
+				obj_ini.gear[target_company][good] = "Psychic Hood"; TODO - consider adding regular human version
+				obj_ini.experience[target_company][good] = 50;
+				obj_ini.race[target_company][good] = 4;
+				unit = new TTRPG_stats("inquisition", target_company, good, "sanctioned_psyker"); TODO - add proper stat lines
+				break;
+			*/
+		// Faction 4.5 - Radical inquisitors, perhaps?
 		// Faction 5 - Sisters of Battle
 			case "Sister of Battle":
-				obj_ini.wep1[target_company][good] = "Bolter"; // Might want to create a "Light Bolter" variant for this one
+				obj_ini.wep1[target_company][good] = "Light Bolter";
 				obj_ini.wep2[target_company][good] = "Sarissa";
-				obj_ini.armour[target_company][good] = "Power Armour"; // Same here, Sororitas are glorified guard
-				obj_ini.experience[target_company][good] = 60;
+				obj_ini.armour[target_company][good] = "Light Power Armour";
+				obj_ini.experience[target_company][good] = 20;
 				obj_ini.race[target_company][good] = 5;
 				unit = new TTRPG_stats("adeptus_sororitas", target_company, good, "sister_of_battle");
 				break;
 			case "Sister Hospitaler":
-				obj_ini.wep1[target_company][good] = "Bolter"; // Same here
+				obj_ini.wep1[target_company][good] = "Light Bolter";
 				obj_ini.wep2[target_company][good] = "Sarissa";
-				obj_ini.armour[target_company][good] = "Power Armour"; // Same here
-				obj_ini.experience[target_company][good] = 100;
+				obj_ini.armour[target_company][good] = "Light Power Armour";
+				obj_ini.experience[target_company][good] = 50;
 				obj_ini.gear[target_company][good] = "Sororitas Medkit";
 				obj_ini.race[target_company][good] = 5;
 				unit = new TTRPG_stats("adeptus_sororitas", target_company, good, "sister_hospitaler");
 				break;
+			/* TODO - if we get to recruiting allied or other leaders, lot of stuff needs to be implemented
+			case "Prioress":
+				obj_ini.wep1[target_company][good] = "Bolter"; TODO - give a cool weapon
+				obj_ini.wep2[target_company][good] = "Sarissa"; TODO - same
+				obj_ini.armour[target_company][good] = "Power Armour"; TODO - Give cool armour
```

Comment:
Can remove Sarissa from them, as it's now built-in into the bolter.
```suggestion
			case "Sister of Battle":
				obj_ini.wep1[target_company][good] = "Light Bolter";
				obj_ini.wep2[target_company][good] = "";
				obj_ini.armour[target_company][good] = "Light Power Armour";
				obj_ini.experience[target_company][good] = 20;
				obj_ini.race[target_company][good] = 5;
				unit = new TTRPG_stats("adeptus_sororitas", target_company, good, "sister_of_battle");
				break;
			case "Sister Hospitaler":
				obj_ini.wep1[target_company][good] = "Light Bolter";
				obj_ini.wep2[target_company][good] = "";
				obj_ini.armour[target_company][good] = "Light Power Armour";
				obj_ini.experience[target_company][good] = 50;
				obj_ini.gear[target_company][good] = "Sororitas Medkit";
				obj_ini.race[target_company][good] = 5;
				unit = new TTRPG_stats("adeptus_sororitas", target_company, good, "sister_hospitaler");
				break;
			/* TODO - if we get to recruiting allied or other leaders, lot of stuff needs to be implemented
			case "Prioress":
				obj_ini.wep1[target_company][good] = "Light Bolter"; TODO - give a cool weapon
				obj_ini.wep2[target_company][good] = ""; TODO - same
```

Reply:
Ok, this should be fixed now.